src/server/src/server/index.ts

```diff
@@ -10,6 +12,22 @@ import logger from '@shared/Logger';
 
 // Start the server
 const port = Number(process.env.PORT || 5000);
-app.listen(port, () => {
+const server = app.listen(port, () => {
     logger.info('Server started on port: ' + port);
 });
+
+const clientPath = 'http://localhost:3000';
```

Comment:
Where should we get the client path? Originally I get from src/app, but it's not working after I merged with main.

Reply:
Add the export keyword to a src/server/src/server/index.ts on line 19. You get it from there then

Reply:
?? This file is src/server/src/server/index.ts. Should I just keep this here?

Reply:
oh oopse sorry. I meant can you move the clientPath variable to src/app/index.ts and then export from there. Also add in in that file's line 24.  `app.use(cors({ credentials: true, origin: clientPath' }));`

Reply:
Done.